File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task158_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    char data[256];
    printf("Enter data: ");
    fgets(data, 256, stdin);
    data[strcspn(data, "
")] = 0; // remove newline character
    char command[256];
    sprintf(command, "echo %s > output.txt", data);
    system(command);
    return 0;
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task158_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter data: ");
        String data = scanner.nextLine();
        try {
            Runtime.getRuntime().exec("echo " + data + " > output.txt");
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}